pkg/log/options.go

```diff
@@ -18,22 +18,34 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
-	"go.uber.org/zap/zapcore"
 )
 
 const (
-	// None is used to disable logging output as well as to disable stack tracing.
-	None zapcore.Level = 100
-
-	defaultOutputLevel        = zapcore.InfoLevel
-	defaultStackTraceLevel    = None
+	defaultOutputLevel        = InfoLevel
+	defaultStackTraceLevel    = InfoLevel
 	defaultOutputPath         = "stdout"
 	defaultErrorOutputPath    = "stderr"
 	defaultRotationMaxAge     = 30
 	defaultRotationMaxSize    = 100 * 1024 * 1024
 	defaultRotationMaxBackups = 1000
 )
 
+// OutputLogLevel is an enumeration of all supported log levels.
+type OutputLogLevel string
```

Comment:
This should be named `OutputLevel` since we're in `package log`. In this package we have context, and outside of this package we refer to it as `log.OutputLevel` so its meaning is clear.

Reply:
Renamed to `Level`. I just realized that `Output` is used to differentiate the stack and output levels, so using just `Level` for both is more appropriate.